app/(company)/work-with-us/components/AshbyCareers.js

```diff
@@ -1,21 +1,31 @@
 'use client';
 import React, { useEffect } from 'react';
 import styles from './AshbyCareers.module.scss';
+import Link from 'next/link';
+import H2 from '@shared/typography/H2';
 
-function AshbyCareers() {
-  useEffect(() => {
-    if (!window.ashbyJobsOptions) {
-      const script = document.createElement('script');
-      script.src = 'https://jobs.ashbyhq.com/goodparty/embed';
-      document.body.appendChild(script);
-    }
-  }, []);
+function AshbyCareers({ jobs }) {
   return (
     <section className={styles.wrapper}>
-      <h2 data-cy="opening-title" className={styles.h2}>
-        Current Openings
-      </h2>
-      <div id="ashby_embed" className="max-w-lg" />
+      <H2 data-cy="opening-title">Current Openings</H2>
+
+      {jobs && jobs.length > 0 && (
+        <div className="grid grid-cols-1 lg:grid-cols-3 gap-6">
```

Comment:
Please keep our existing pattern for grids - always use grid-cols-12 and set the cols on the children.